platform/mellanox/mlnx-platform-api/sonic_platform/eeprom.py

```diff
@@ -25,6 +26,18 @@
 #
 EEPROM_SYMLINK = "/var/run/hw-management/eeprom/vpd_info"
 
+platform_name = get_platform()
+if 'simx' in platform_name:
+    if is_host():
```

Comment:
@tomer-israel why not to use `get_path_to_platform_dir`?

Reply:
i used get_path_to_platform_dir() instead